geos/meta.yaml

```diff
@@ -8,8 +8,7 @@ source:
     - cmake.win.patch  # [win]
 
 build:
-  number: 3  # [win]
-  number: 2  # [not win]
+  number: 3
```

Comment:
ooo, ouch. Not sure this compiles in time for the CI services does it? I always thought we would have to build this by hand when it came to GEOS.